internal/app/networkd/pkg/address/static.go

```diff
@@ -16,8 +16,10 @@ import (
 
 // Static implements the Addressing interface
 type Static struct {
-	Device      *machine.Device
+	CIDR        string
+	Mtu         int
```

Comment:
This is done as when you have VLANs under the machine.Device that has their own parameters we cannot use the machine.Device to set the CIDR. 

This could probably be refactored to something else but the base concept of the machine.Device need to change in order to support the idea of VLANs as subdevices.